ui/test/nuxeo-video-conversions.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, flush, html, isElementVisible } from '@nuxeo/nuxeo-elements/test/test-helpers';
+import '../nuxeo-video/nuxeo-video-conversions.js';
+import videoProperties from './resources/videoProperties';
+/* eslint-disable no-unused-expressions */
+
+suite('nuxeo-video-conversions', () => {
+  suite('Displaying element when no parameters provided', () => {
+    let element;
+
+    setup(async () => {
+      element = await fixture(html`
+        <nuxeo-video-conversions></nuxeo-video-conversions>
+      `);
+      await flush();
+    });
+
+    test('Should not display the label when label is not provided', async () => {
+      expect(isElementVisible(element.root.querySelector('h3'))).to.be.false;
+    });
+
+    test('Should not display any other element when no document provided', async () => {
+      expect(isElementVisible(element.shadowRoot.querySelector('.flex'))).to.be.false;
+    });
```

Comment:
These methods are not `async` anymore since the `flush`, that raised the need for awaiting, was moved to the setup.